css-view-transitions-1/Overview.bs

```diff
@@ -512,6 +514,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 	- a [=backdrop root=].
 
+## Interactions during suppressed rendering ## {#interactions-during-suppressed}
+
+	While a {{Document}} |document|'s [=document/transition suppressing rendering=] is true,
+	pointer hit testing must target |document|'s [=document element=],
+	ignoring all other {{Element}}s and [=pseudo-elements=].
+
+	Note: This does not effect pointers that are [=pointer capture|captured=].
```

Comment:
Waiting on https://github.com/w3c/pointerevents/pull/470 so I can reference this term.